spinalcordtoolbox/deepseg/models.py

```diff
@@ -342,6 +351,15 @@
                              'the Canadian Prospective Cohort Study (CanProCo).',
          'url': 'https://github.com/ivadomed/canproco',
          'models': ['model_seg_sc_lesion_canproco_nnunet']},
+    'seg_sc_epi':
+        {'description': 'Spinal cord segmentation for EPI data',
+         'long_description': 'This segmentation model for spinal cord on EPI data uses a 3D nnUNet model from '
+                             'the nnUNetv2 framework. The training data consists of 3D images (n=192) spanning numerous resolutions '
```

Comment:
Is it worth mentioning whether these are healthy controls or pathological?

Reply:
Thanks for the feedback. Made the required changes.